modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -37,6 +27,74 @@ Datacenter: Atlanta
 |openshift-region: us-east-1 (tag), openshift-zone: us-east-1b (tag)
 |This defines a different failure domain within the same region called us-east-1b.
 |===
+
+== Creating vSphere storage topology during installation or post-installation
+
+=== Procedure
+. In the VMware vCenter vSphere client GUI, define appropriate zone and region catagories and tags.
++
+While vSphere allows you to create categories with any arbitrary name, {product-title} strongly recommends use of `openshift-region` and `openshift-zone` names for defining topology.
++
+For more information about vSphere categories and tags, see the VMware vSphere documentation.
+
+. In {product-title}, create failure domains:
+* xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#installation-installer-provisioned-vsphere-config-yaml_installing-vsphere-installer-provisioned-customizations[During installation]
```

Comment:
This should link to - https://56145--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html#configuring-vsphere-regions-zones_installing-vsphere-installer-provisioned-network-customizations 

Which actually points to the document which is about zones and region configuration during installation.

Reply:
Done.